### ISSUE TITLE
Incorrect work after long stay

### ISSUE THREAD

**Issue by ShGKme** [nextcloud/talk-desktop#37]
## Steps to reproduce

1. Open a chat
2. Close a laptop, go to sleep mode
3. Wait ~10 hours
4. Open a laptop
5. Try to use the application

## Expected behavior

It works as yesterday

## Actual behavior

- Infinity connecting to a new chat
- Selected chat in the chat list and currently opened chat (incl. chat history) are not equal
- New windows are opened themself

## Note

Refreshing the page helps

## May be related

Sometimes I see these errors on the Talk web after a long stay. It might be related.

![image](https://user-images.githubusercontent.com/25978914/224969971-a8405c18-2b4a-4f59-bc7f-3790814d7731.png)

**Comment by AWagnr** [comment]
I'm experiencing this behavior every day. Any update on this?

**Comment by nickvergessen** [comment]
You can close the desktop client and restart it as a workaround

**Comment by AWagnr** [comment]
yes I know, it's just a bit annoying ;-)

**Comment by AWagnr** [comment]
@ShGKme I disagree. I'm still experiencing this, especially I cannot join calls after resuming from sleep after a certain period of time. There is an infinite loop trying to join. It only works after close / reopening the desktop client.
